ethers-signers/src/ledger/mod.rs

```diff
@@ -25,6 +25,11 @@ impl Signer for LedgerEthereum {
 
     /// Signs the transaction
     async fn sign_transaction(&self, message: &TypedTransaction) -> Result<Signature, Self::Error> {
+        let mut tx_with_chain = message.clone();
```

Comment:
is this value even used?
because we're still using `message`: `self.sign_tx(message).await`